products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,73 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision, and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users globally. It is very important that, not only the main pages are localized, they are thoroughly tested before they are launched on production.  
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/
+* Pontoon: https://pontoon.mozilla.org/projects/mozillaorg/
+* Locamotion: https://mozilla.locamotion.org/projects/mozilla_lang/
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
```

Comment:
Wrong link + typo, should be `https://l10n.mozilla-community.org/webdashboard/?locale={locale code}`